server/.env.example

```diff
@@ -9,3 +9,6 @@ ALERTMANAGER_CONTAINERPORT=9093
 # grafana HostPort and ContainerPort
 GRAFANA_HOSTPORT=1919
 GRAFANA_CONTAINERPORT=3000
+
+# choose language, en or cn, default en
```

Comment:
change to `set Grafana dashboard language, could be en or cn,  using en by default`